### PR TITLE
Support cert-manager v0.15.2, replace deprecated gotransip v5 in favor of v6

### PR DESCRIPTION
**Pull request by danielmorlock**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -113,17 +122,19 @@ func (c *transipDNSProviderSolver) Present(ch *v1alpha1.ChallengeRequest) error
...
+	for _, s := range dnsEntries {
 		if s == acmeDnsEntry {
+			fmt.Printf("ACME DNS entry already exists, skip\n")
 			return nil
 		}
 	}
 
 	// Record does not exist - append it to the existing DNS entries
-	domainInfo.DNSEntries = append(domainInfo.DNSEntries, acmeDnsEntry)
+	dnsEntries = append(dnsEntries, acmeDnsEntry)
```
Comment (robbietjuh) [inline main.go]: This line has become redundant now that `gotransip` has a `AddDNSEntry` function. Its only use was to compose the complete set of DNS records, as the complete set had to be passed on to TransIP.

Could you remove this line?

**Review by robbietjuh — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you have a look at the comment in `main.go`? Should be a simple fix :)

File: main.go (hunk 1 above)
Comment (danielmorlock) [reply main.go]: You're right. Removed.

**Review by robbietjuh — APPROVED** [review]
Awesome, thank you!

**Pull request merged**